lib/rubycritic/generators/console_report.rb

```diff
@@ -0,0 +1,52 @@
+require "colorize"
+
+module Rubycritic
+  module Generator
+    class ConsoleReport
+      def initialize(analysed_modules)
+        @analysed_modules = analysed_modules
+      end
+
+      def generate_report
+        reports = @analysed_modules.sort.map do |analysed_module|
+          generate_module_report analysed_module
+        end
+        puts reports.join("\n")
+      end
+
+      private
+
+      def generate_module_report(mod)
+        report = <<-EOF
```

Comment:
I know why you're doing this but I still find it very detrimental for readilbility. I'd rather use something like [here](http://rubyquicktips.com/post/4438542511/heredoc-and-indent)